osu.Server.Queues.ScoreStatisticsProcessor.Tests/DatabaseTest.cs

```diff
@@ -58,17 +58,32 @@ protected DatabaseTest()
                 if (db.QueryFirstOrDefault<int?>("SELECT * FROM osu_counts WHERE name = 'is_production'") != null)
                     throw new InvalidOperationException("You are trying to do something very silly.");
 
+                try
+                {
+                    // Until osu-web adds the alias views, let's take things into our own hands.

```

Comment:
osu-web already added the [aliases](https://github.com/ppy/osu-web/blob/master/database/migrations/2023_10_31_074205_create_score_table_aliases.php)

Reply:
Yeah, I was led astray from the error message when running `TRUNCATE` (table doesn't exist). Have removed this.